### ISSUE TITLE
A11Y: Display list of navigational shortcuts

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#4243]
## Feature Description

Upon detecting* that a user is using a keyboard to navigate the application, show a dialog with navigational shortcuts.

* Probably upon detecting the user tabbing through the application before using a mouse (similarly to slack). To be defined.

@samitron7 to define UX.

@pbakaus to set priority.

## Additional Context

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by swissspidy** [comment]
So this would be like the message you see when tabbing on Google Docs (or was it another app)?

**Comment by swissspidy** [comment]
@pbakaus you mentioned an existing thread about this topic if I am not mistaken?

**Comment by o-fernandez** [comment]
Per @samitron7 they decided use just one display list. This separate list for navigational shortcuts are not needed.
